docs/changelog/2024.7.0.rst

```diff
@@ -93,8 +93,8 @@ Backwards Incompatible Changes
 Bug Fixes
 ---------
 
-- - Enabled |validate_quantities| to be compatible with postponed evaluation of
-    annotations (see :pep:`563`). (:pr:`2479`) (:pr:`2506`)
+- Enabled |validate_quantities| to be compatible with postponed evaluation of
+  annotations (see :pep:`563`). (:pr:`2479`, :pr:`2506`)
```

Comment:
The formatting here was weird. It showed up as a bullet point within a bullet point.